pygeos/_geom.pyx

```diff
@@ -0,0 +1,82 @@
+from cpython cimport PyObject
```

Comment:
Minor nitpick, but maybe we can name this file `_geometry.pyx` instead of `_geom.pyx`, so it keeps the name of the python file to which it belongs (the cython algos here are publicly exposed in python in `geometrypy`)?